enrichissement.py

```diff
@@ -125,6 +128,46 @@ def manage_column_final(df):
                              'geolocCommuneEtablissement'])
     return df
 
+  
+
+
+#Enrichir avec le departement des acheteur et des etablissements. Base departement francais.
+def extraction_code_postal(code_postal):
+    """renvoie le code postal en prenant en compte les Drom
+    code_postal est un str"""
+    try:
+        code = code_postal[:2]
+        if code == "97" or code == "98":
+            code = code_postal[:3]
+        return code
+    except:
+        return "00"
+
+#Enrichir avec le departement des acheteur et des etablissements. Base departement francais.
+def enrichissement_departement(df):
+    path = os.path.join(path_to_data,conf["departements-francais"])
+    departement = pd.read_csv(path, sep = "\t")
+    sub_departement = departement[['NUMÉRO', 'NOM', 'REGION']]
+    #codePostalAcheteur pour le departement de l acheteur
+    #codePostalEtablissement pou l'Etablissement
+    #Creation de deux variables récupérant le numéro du departement
+    df["departementAcheteur"] = df.apply(lambda x: extraction_code_postal(x["codePostalAcheteur"]), axis=1)
```

Comment:
pas besoin de passer par une lambda :
```suggestion
    df["departementAcheteur"] = df["codePostalAcheteur"].apply(extraction_code_postal)
```